src/test/java/org/broadinstitute/hellbender/tools/SplitNCigarReadsIntegrationTest.java

```diff
@@ -53,4 +53,15 @@ public void testSplitsFixNDN() throws Exception {
                 Arrays.asList(getTestDataDir() +"/" + "expected.splitNCigarReadsSnippet.splitNcigarReads.fixNDN.bam"));
         spec.executeTest("test fix NDN", this);
     }
+
+    /**
+     * Currently disabled due to issue: https://github.com/broadinstitute/gatk/issues/1865
+     */
+    @Test(enabled = false) //regression test for https://github.com/broadinstitute/gatk/pull/1853
```

Comment:
This does fix it. However, is there a reason not to also change ReadCoordinateComparator::compareCoordinates to take the case when an unmapped read's mate is mapped? That would also fix this particular problem.